networkx/algorithms/operators/binary.py

```diff
@@ -274,17 +297,25 @@ def compose(G, H):
 
     Returns
     -------
-    C: A new graph  with the same type as G
+    C: A new graph with the same type as G
+
+    See Also
+    --------
+    Graph.update
```

Comment:
This is great! Thanks!

Just a tiny nit pick here. `Graph.update` wouldn't automatically link this function to the right place in the documentation. You need to use something like `` :func:`~networkx.Graph.update` `` so sphnix can pick up the right place to link the update function.

Reply:
this is how it looks right now https://output.circle-artifacts.com/output/job/35a2e7cc-df57-4836-aca8-c48d2d29bbbd/artifacts/0/doc/build/html/reference/algorithms/generated/networkx.algorithms.operators.binary.union.html#networkx.algorithms.operators.binary.union

Reply:
Thanks for catching the broken link @MridulS .   And for showing me how to reference functions that reside elsewhere. 

I have fixed the Graph.update() link in the three "See Also" sections where it is used. A new PR is on its way.